diesel/src/query_builder/ast_pass.rs

```diff
@@ -207,7 +210,7 @@ where
     pub fn push_bind_param<T, U>(&mut self, bind: &'b U) -> QueryResult<()>
     where
         DB: HasSqlType<T>,
-        U: ToSql<T, DB>,
+        U: ToSql<T, DB> + ?Sized,
```

Comment:
Can you explain the reasoning for `?Sized`

Reply:
That's required so that we can have something like `AstPass:push_bind_param::<Text, str>(…, "abc")` as `str` is not `Sized` there. So this requirement relaxes the bounds for this type, by allowing more types there.